File: src/DashboardUI/src/data-contracts/WaterRightInfoListItem.ts
export interface WaterRightInfoListItem {
  allocationUuid: string;
  waterRightNativeId: string;
  owner: string;
  priorityDate: Date;
  expirationDate: Date;
  legalStatus: string;
  flow: number;
  volume: number;
  beneficialUses: string[];
}

File: src/API/WesternStatesWater.WestDaat.Contracts.Client/WaterRightInfoListItem.cs
namespace WesternStatesWater.WestDaat.Contracts.Client
{
    public class WaterRightInfoListItem
    {
        public string AllocationUuid { get; set; }
        public string WaterRightNativeId { get; set; }
        public string Owner { get; set; }
        public DateTime? PriorityDate { get; set; }
        public DateTime? ExpirationDate { get; set; }
        public string LegalStatus { get; set; }
        public double? Flow { get; set; }
        public double? Volume { get; set; }
        public List<string> BeneficialUses { get; set; }
    }
}
